examples/eks_cluster_custom_iam/iam.tf

```diff
@@ -63,6 +63,11 @@ resource "aws_iam_role" "assume_role" {
         Principal = {
           AWS = data.castai_eks_user_arn.castai_user_arn.arn
         }
+        Condition = {
+          StringEquals = {
+            "sts:ExternalId" = data.castai_eks_clusterid.castai_cluster_id.id
```

Comment:
Cool feature of AWS is this externalID :)
https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html